babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
- [ ] I kindly suggest that you set up your babel.config.js as shown in this [tutorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)